tests/misc/tstreamsreadline.nim

```diff
@@ -0,0 +1,27 @@
+discard """
+  output: ""
+"""
```

Comment:
we have these, why not name it tests/stdlib/tstreams4.nim
```
tests/stdlib/tstreams.nim
tests/stdlib/tstreams2.nim
tests/stdlib/tstreams3.nim
```

Reply:
Because my search didn't yield those :) Fixed.